code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -41,6 +41,12 @@
 		active_hotspot.just_spawned = (current_cycle < SSair.times_fired)
 			//remove just_spawned protection if no longer processing this cell
 		SSair.add_to_active(src, 0)
+	var/list/adjacent_dense_turfs = AdjacentTurfs(FALSE, TRUE, TRUE) //Checks adjacent  turfs for non-dense turfs, only in the 4 cardinal directions
```

Comment:
For non-dense turfs? Why is it called `adjacent_dense_turfs` if so? (I think the comment is wrong)

Reply:
Also there's an extra space in `adjacent  turfs`